examples/short_example_synthetic_control.py

```diff
@@ -0,0 +1,56 @@
+from itertools import product
+
+import numpy as np
+import pandas as pd
+
+from cluster_experiments.experiment_analysis import SyntheticControlAnalysis
+from cluster_experiments.perturbator import ConstantPerturbator
+from cluster_experiments.power_analysis import PowerAnalysisWithPreExperimentData
+from cluster_experiments.random_splitter import FixedSizeClusteredSplitter
+
+
+def generate_data(N, start_date, end_date):
+    # Generate a list of dates between start_date and end_date
+    dates = pd.date_range(start_date, end_date, freq="d")
+
+    users = [f"User {i}" for i in range(N)]
+
+    # Use itertools.product to create a combination of each date with each user
+    combinations = list(product(users, dates))
+
+    target_values = np.random.normal(0, 1, size=len(combinations))
+
+    df = pd.DataFrame(combinations, columns=["user", "date"])
+    df["target"] = target_values
+
+    # Ensure 'date' column is of datetime type
+    df["date"] = pd.to_datetime(df["date"])
+
+    return df
+
+
+# Example usage
+
+
+if __name__ == "__main__":
+
+    df = generate_data(10, "2022-01-01", "2022-01-30")
+
+    sw = FixedSizeClusteredSplitter(n_treatment_clusters=1, cluster_cols=["user"])
+
+    perturbator = ConstantPerturbator(
+        average_effect=0.1,
+    )
+
+    analysis = SyntheticControlAnalysis(
+        cluster_cols=["user"], time_col="date", intervention_date="2022-01-15"
+    )
+
+    pw = PowerAnalysisWithPreExperimentData(
+        perturbator=perturbator, splitter=sw, analysis=analysis, n_simulations=50
+    )
+
+    power = pw.power_analysis(df)
+    print(f"{power = }")
+    print(pw.power_line(df, average_effects=[0.1, 0.2, 0.5, 1, 1.5], n_jobs=-1))
+    print(pw.simulate_point_estimate(df))
```

Comment:
adding it here because can't add in notebook. you have an extra parenthesis in this line:
```python
pl = pw.power_line(df, average_effects=[0.1, 0.2, 0.5, 1, 1.5], n_jobs=-1))
```

Reply:
also, in the notebook we don't seem to have power = alpha for effect = 0. do you know why might this happen?

Reply:
can you also add some simulations of point estimates in the notebook? so users can see that the estimate is unbiased?

Reply:
Is this true even when the groups aren't equal? I have some weird results using OLS for 1 cluster treatment and the rest in control (power around 70%)

Reply:
ah, maybe the variance explodes with 1 treatment cluster

Reply:
if you try:
```python
sw = FixedSizeClusteredSplitter(n_treatment_clusters=5, cluster_cols=["user"])
```
for ols, do you have the same issue?

Reply:
btw does synthethic only work with n_treatment_clusters=1?

Reply:
yes, it only work with 1 treatment cluster. I couldn't figure out an easy way to run several treatment clusters. 

Basically,  we need to find the best combination of weights for 1 treatment cluster at a time. So this means we need to run the model sequentially, for every treatment cluster. However, we need to store somewhere which are the treatment cluster because they can't be used as donors.